README.md

```diff
@@ -356,7 +356,7 @@ make docker-build-registry
 make cluster-up
 
 # bridge up a local cluster with kubernetes 1.25
```

Comment:
please change this comment / either to 1.30 or just making it generic

Reply:
DONE